novawallet/Common/Substrate/Calls/Multisig/Multisig+Call.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import SubstrateSdk
+import BigInt
+
+extension Multisig {
+    struct AsMultiCall<Weight: Codable>: Codable {
+        enum CodingKeys: String, CodingKey {
+            case threshold
+            case otherSignatories = "other_signatories"
+            case maybeTimepoint = "maybe_timepoint"
+            case call
+            case maxWeight = "max_weight"
+        }
+
+        @StringCodable var threshold: UInt16
+        let otherSignatories: [AccountId]
```

Comment:
It should be a list of `BytesCodable` otherwise it will not work on some networks like evm